migrations/sqls/20190324132907-dbdev-up.sql

```diff
@@ -0,0 +1,185 @@
+/* Replace with your SQL commands */
+/*
+Create users tables: I figured that later on, this application can be scaled to mainain, volunteers, board members, and maybe even donors and staff (HR management) 
+*/
+CREATE TABLE IF NOT EXISTS users (
+  userId int AUTO_INCREMENT PRIMARY KEY,
+  firstName varchar(64) NOT NULL,
+  lastName varchar(64) NOT NULL,
+  accessLevel ENUM('Administrator','Manager','Staff'), 
+  userName nvarchar(64) NOT NULL,
+  password nvarchar(64) NOT NULL
+);
+INSERT INTO users(firstName,lastName,accessLevel,userName,password)
+VALUES('Leonel','Nieto','Administrator','leonelnieto','password');
```

Comment:
We shouldn't put this into github -- would give away the password to anyone who looks at this repo. Instead, you can insert into your local and production database your admin user account by following [these instructions in the readme](https://github.com/JustUtahCoders/comunidades-unidas-internal#connecting-to-the-database)